src/main/java/de/tum/in/www1/exerciseapp/web/rest/QuizExerciseResource.java

```diff
@@ -55,7 +59,27 @@ public QuizExerciseResource(QuizExerciseRepository quizExerciseRepository, Parti
         if (quizExercise.getId() != null) {
             return ResponseEntity.badRequest().headers(HeaderUtil.createFailureAlert(ENTITY_NAME, "idexists", "A new quizExercise cannot already have an ID")).body(null);
         }
+
```

Comment:
Methods in this class generally get quite long and complex. Please talk to @MoritzIssig about restructuring these methods (e.g. by extracting code into other methods, avoiding code duplication, etc.) in a logical and useful way. I suggest you do that after the 2 currently open (and huge) pull requests get merged, to avoid complex merge conflicts, so please add it as TODO into JIRA.

Reply:
I have added the task in JIRA